nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const app = express();
+
+app.use(function (req, res, next) {
+  console.log('Time:', Date.now());
+  next();
+}, function (req, res, next) {
+  console.log('Request Type:', req.method);
+  next();
+});
```

Comment:
@gizemcandemir, As I commented on pull request by @sowmya1408, you can log both the HTTP method and time within the same callback function.
```js
app.use((req, res, next) => {
    console.log("method " + req.method);
    console.log("date " + Date.now());
    next();
});
```
You can also do it using just one `console.log` statement by using [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
```js
app.use((req, res, next) => {
    console.log(`method: ${req.method}\ndate: ${Date.now()}`);
    next();
});
```